priv/repo/migrations/20170730233801_add_role_to_user.exs

```diff
@@ -12,6 +12,6 @@ defmodule Api.Repo.Migrations.AddRoleToUser do
 
     flush()
 
-    from(u in "users", update: [set: [role: "participant"]]) |> Api.Repo.update_all([])
+    from(u in "users", update: [set: [role: "participant"]]) |> ApiWeb.Repo.update_all([])
```

Comment:
Why not comment out altogether? That seems to be where data migrations end up, anyway.

Reply:
I only commented the ones that failed. But you're right, it makes sense to do it.